src/taskEvents/logCard.js

```diff
@@ -0,0 +1,170 @@
+import { createElement } from './utils.js';
+
+function createSummarySection(username, log, isAllTasks, createModalFunction) {
+  const summary = createElement({
+    type: 'summary',
+    attributes: { class: 'summary' },
+  });
+
+  const summaryContainer = createElement({
+    type: 'div',
+    attributes: { class: 'summary-container' },
+  });
+
+  if (isAllTasks) {
+    const name = createElement({
+      type: 'button',
+      attributes: { class: 'name' },
+      innerText: username,
+    });
+    name.addEventListener('click', () => createModalFunction(username));
+    summaryContainer.appendChild(name);
+  }
+  const tasklog = createElement({
+    type: 'p',
+    attributes: { class: 'log' },
+    innerText: log,
+  });
+  const icon = createElement({
+    type: 'img',
+    attributes: {
+      class: 'dropDown',
+      src: './assets/down.png',
+      alt: 'dropdown icon',
+    },
+  });
+
+  summaryContainer.appendChild(tasklog);
+  summary.appendChild(summaryContainer);
+  summary.appendChild(icon);
+
+  return summary;
+}
+
+function createDetailsSection(title, purpose, category, level) {
```

Comment:
It would be good if we can wrap these arguments as an object so that they could be passed in any order

Reply:
okay will do it